versions/3.0.md

```diff
@@ -1128,20 +1127,38 @@ A request body with a referenced model definition.
       "schema": {
         "$ref": "#/components/schemas/User"
       },
-      "examples": [ "http://foo.bar/examples/user-example.json" ]
+      "examples": {
```

Comment:
I don't understand the value.  Since `examples` is no longer an array if items, what is the significance of the map key?